backend/app/services/repositories/user_repository.py

```
@@ -0,0 +1,90 @@
+
+    def __del__(self):
+        if hasattr(self, "_db_gen") and self._db_gen is not None:
+            try:
+                self._db_gen.close()
+            except Exception:
+                pass
+
+    def get_user_by_email(
+        self,
+        email: str,
+        attributes: list[str] = [],
```

Ruff (0.14.10)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)